functions.php

```diff
@@ -0,0 +1,36 @@
+<?php
+require "connectToDb.php";
+
+/*
+ * function getArtworks
+ * returns an array of the artworks
+ */
+
+function getArtworks() : array
+{
+    $db = getDB();
+    $query = $db->prepare('SELECT * FROM `artworks`;');
+    $query->execute();
+    return $query->fetchAll();
+}
+
+/*
+ * function displayArtworks
+ * @params array
+ * returns HTML displaying the artwork details
+ */
+
+function displayArtworks(array $data) : string
+{
+    $return = '';
+    foreach ($data as $artwork){
+        $return .= '<div class="artworks">';
+        $return .= '<img src="' . $artwork['image'] . '" height="250" />';
```

Comment:
Might be worth checking to make sure there is an image in the db before adding the img tag, otherwise you end up with broken images.